### ISSUE TITLE
Opaque/Transparent Leaves

### ISSUE THREAD

**Issue by ghost** [TeamTwilight/twilightforest#252]
Forge version: 2512
Twilight Forest version: (this is the build number) 3.1.128
Link to crash log: (please use a paste site, do not attach the .txt or paste the log inline)
No Crash.

Steps to reproduce:
1. Create a Twilight Dimension and look at the trees leaves

Im using Optifine for MC 1.12.2 and have set it to use Opaque leaves instead of Transparent to try and increase my FPS. TwilightForest trees seem to use Transparent leaves.  Could you adapt TwilightForest to respect this option? My resource pack is ChromaHills which has Opaque and Transparent Leave Textures.

So basically have an option for Transparent or Opaque leaves?

Thanks

**Comment by Andromander** [comment]
At the moment, leaves can be set to opaque via Fast Graphics. However, an option to set opaque leaves via Optifine may not be possible due to several existing issues with TF and Optifine. The devs may know more

**Comment by williewillus** [comment]
we can't do anything other than respect the vanilla fast/fancy option (which we do currently). Optifine probably has its own separate flag, but we obviously can't access that.

I'd suggest switching vanilla to fast then selectively reenabling fancy things in the optifine menus

**Comment by ghost** [comment]
so theres absolutely no way you can have your own config option in TwilightForest to generate your own Transparent/Opaque leaves?   I was just thinking about players FPS rates.

Thanks for answering anyways.
